### ISSUE TITLE
Arquitetura de Informação / Definição de objetivos do site do I/O Talks.

### ISSUE THREAD

**Issue by ViniciusVinna** [valadares-ti/site#3]
# Briefing:

A página do I/O Talks (io-talks.org) será o site-matriz de todas as edições do evento com as informações gerais da comunidade, notícias, informações sobre os próximos eventos, mídia-kit, press-release e precisa oferecer todo o material de suporte para a criação de outras edições, além do programa de patrocínio.
- **Ponto-chave** - Podemos trabalhar com um _WordPress MU_, onde o site matriz terá um tema principal, e os eventos regionais (gv.io-talks.org ou io-talks.org/gv) terão um tema landing-page padrão com estrutura de _single-page_ (vitrine, dia, local e tema/trilha, botão de inscrição, palestrantes, cronograma, patrocínio/apoio, depoimentos, manifesto e organizadores.
# Estrutura I/O Talks (Matriz):
- **Home** - Sugestões...
- **Eventos** - Informações com os próximos eventos, com links para landing page, local e data.
- **Cursos e Workshops** - Informações sobre cursos e workshops ministrados pelos membros organizadores do I/O Talks. A ideia é criar uma landing-page padrão também para facilitar a criação.
- **Conteúdo** - Área destinada a produção de organização de conteúdos específicos: E-books, Infográficos, Webinars, Screencasts e slides de palestras.
- **Patrocinadores** - Boxes com a marca e descrição breve das empresas e informações sobre como se tornar um patrocinador, com informações e métricas segmentadas da comunidade e dos eventos.
- **Newsletter** - 
- **Imprensa** - Mídia kit, Manual da marca, notícias sobre o evento em outros veículos.
# Estrutura I/O Talks Regional (Landing Page):
- **Header** - Data com local do evento, breve descrição.
- **Botões call-to-action** - fazer Inscrição, seja um patrocinador.
- **Trilha do evento** - Opensource, Microsoft, Game development etc...
- **Palestrantes** - Nome, cargo, empresa, links twitter, linkedin e github.
- **Programação/Agenda** - Horários -> Tema da palestra e informações do palestrante.
- **Fazer inscrição** - Botão, área com mais informações sobre o evento.
- **Patrocinadores** - Boxes com a marca e descrição breve das empresas.
- **Local** - Mapa do Google Maps com a localização do evento.
- **Organizadores e Apresentadores** - Foto, Nome, Links para redes sociais e descrição rápida.
- **Link para Código de conduta**
# Sugestões:
- Para aumentar as métricas da comunidade, seria interessante os usuários fazerem algum tipo de cadastro inicial (via facebook) para segmentarmos melhor o tamanho da comunidade por área, e para aumentar a relevância para futuros patrocinadores e apoiadores.
- Todos os eventos serão organizados e padronizados no Eventick.
- Redes sociais - Facebook, Twitter, Instagram e Youtube.
- Todas as edições do evento serão filmadas integralmente e disponibilizadas no canal do Youtube.

**Comment by janynnegomes** [comment]
Ainda não entendi, isso é pra ser uma comunidade ou um evento?

**Comment by ViniciusVinna** [comment]
Evento e comunidade, vamos pregar a palavra do T.I. e montar a igreja do T.I. Amém?

**Comment by joelgarciajr84** [comment]
As duas coisas, o evento é um dos frutos, mas a árvore é a comunidade.

**Comment by janynnegomes** [comment]
Pelo que entendo @joelgarciajr84 e @ViniciusVinna , a comunidade tem em uma das ramificações os eventos como coding dojo, meet ups pra networking, eventos anuais locais, regionais e nacionais e etc, mas o evento não é o centro de tudo. 

Ex: **Wordcamp** é apenas um evento, que se propagou, mas a comunidade por trás dele é a **WordPress Brasil**(no caso do Brasil), que tem conteúdo relevante e robusto, que contribui em fóruns, que tem toda um estrutura de comunidade, que contribui com documentação e etc, portanto penso eu que o ideal é criar o site da comunidade, com a cara de uma comunidade, sempre focado na oferta de conteúdo de aprendizado (artigos, vídeos, tutoriais e etc) tanto quanto na divulgação dos encontros periódicos e tal.

**Comment by henrich-m** [comment]
@devnetgomez :+1: Concordo em gênero numero e grau. Por mais bacana que seja pensar em uma estrutura de eventos desse porte ( regional ?! a comunidade não é para GV ? ), acho que deveríamos primeiro em focar na comunidade em SI e o que podemos fazer para fomentar ela.

**Comment by janynnegomes** [comment]
Só pra exemplificar o que eu quero dizer pessoal, esse site é feio, mas tem a estrutura de uma comunidade: fórum, artigos, tutoriais e etc. Tem até loja que vende peças pros projetos lá dentro -> http://labdegaragem.com

**Comment by joelgarciajr84** [comment]
:confused:  Então que rumo tomar, quais as sugestões?

**Comment by henrich-m** [comment]
KISS <- Keep It Simple Stupid e Focar no que realmente importa, criar uma comunidade de profissionais de TI, que produza conteúdo e afin, que agregue mais valor do que companhia para cerveja.

**Comment by joelgarciajr84** [comment]
Então o melhor agora é focar aqui https://github.com/io-talks/wp-site-theme , certo?

**Comment by henrich-m** [comment]
@joelgarciajr84 ++

**Comment by ViniciusVinna** [comment]
Concordo, mas acho que é tranquilo preparar o terreno para outras edições, até por que meu irmão já organiza um evento de CSS em Vitória.  Porém sobre a comunidade local, nós já tivemos a experiência de criar grupos no Facebook para fomentar a integração, o que não deu muito certo. O que eu observei é que um dos motivos dos integrantes de comunidade local, não interagem entre si é para não expor suas limitações com medo de serem julgados, e por medo de perderem seus empregos, por estímulo de concorrência. Esse tipo de evento em capital, funciona na mecânica de "comunidade converge para eventos", pelo tamanho, se você tem uma comunidade com 50.000 membros e faz um evento onde 200 vão, seu evento é um sucesso. A nossa comunidade é infinitamente menor e menos engajada, a gente tem que lhe dar com essa situação e identificar esse nosso público-alvo para que nós não percamos o tesão. Como a Univale cedeu espaço do auditório, estrutura técnica, e espaço de mídia (portal) podemos utilizar isso para criar um evento bimestral talvez, com palestrantes locais, no esquema de trilhas para fazer a comunidade convergir e integrar outras áreas que não só sistemas de informação, profissional de infra (manutenção, redes), Design, Marketing Digital. E nesses intervalos entre os eventos a gente gera conteúdo para comunidade, webinar, o @marcelobarbosadg é designer e pode criar infográfico, @joelgarciajr84 e @devnetgomez já tem costume de gerar conteúdo em seus blogs. Acho que dá para convergir tudo que todo mundo faz em um lugar só.

**Comment by ViniciusVinna** [comment]
@joelgarciajr84 temos que definir essa questão primeiro, para revisar a estrutura do site e o conteúdo.

**Comment by janynnegomes** [comment]
Pessoal, na issue #4 eu rascunhei meus pensamentos sobre organização da comunidade, se puderem dar uma olhada.

@ViniciusVinna talvez não tenha dado certo o grupo no Facebook pq as pessoas queriam conteúdo, queriam que algum profissional chegasse até eles e fosse guiando o aprendizado, nós sabemos que nas universidades o ensino acadêmico é bem teórico e conservador, o que na maioria dos casos desestimula pessoas com potencial.

O que eu vejo por aí é que muitos grupos de Facebook visam apenas fazer propaganda, mas nada de ajudar os caras, nada de tirar dúvidas, pq realmente não é um espaço adequado pra isso. Vejo também postagens de links, puros, sem nenhum comentário tecido pela pessoa que postou, explicando o motivo da postagem e em quê aquilo vai ajudar os membros.

Por isso defendo a idéia de uma plataforma de comunidade pra esse fim., pois é um lugar propício, é um lugar onde quem chega tem interesse e vamos prover formas dela voltar e ainda gerar conteúdo.

Creio que num primeiro momento, nós profissionais experientes devamos montar o conteúdo, pra que quando eles chegarem, encontrem motivos pra voltar e não perder o **"tesão"**, como você mesmo falou.
